src/pt/mangalivre/src/eu/kanade/tachiyomi/extension/pt/mangalivre/MangaLivre.kt

```diff
@@ -1,17 +1,118 @@
 package eu.kanade.tachiyomi.extension.pt.mangalivre
 
+import android.app.Application
+import android.content.SharedPreferences
+import android.widget.Toast
+import androidx.preference.EditTextPreference
+import androidx.preference.PreferenceScreen
 import eu.kanade.tachiyomi.multisrc.madara.Madara
+import eu.kanade.tachiyomi.network.POST
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.ConfigurableSource
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import okhttp3.FormBody
+import okhttp3.Response
+import org.json.JSONArray
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import uy.kohesive.injekt.Injekt
+import uy.kohesive.injekt.api.get
+import java.io.IOException
 import java.text.SimpleDateFormat
 import java.util.Locale
 
-class MangaLivre : Madara(
-    "Manga Livre",
-    "https://mangalivre.tv",
-    "pt-BR",
-    SimpleDateFormat("MMMM dd, yyyy", Locale("pt")),
-) {
+class MangaLivre :
+    Madara(
+        "Manga Livre",
+        "https://mangalivre.tv",
+        "pt-BR",
+        SimpleDateFormat("dd.MM.yyyy", Locale("pt")),
```

Comment:
Not using any locale specific patterns,
```suggestion
        SimpleDateFormat("dd.MM.yyyy", Locale.ROOT),
```